tapdb/sqlc/migrations/000026_asset_group_version_customsubtree.up.sql

```diff
@@ -0,0 +1,31 @@
+-- Add version field to table asset_groups.
+ALTER TABLE asset_groups ADD COLUMN version INTEGER NOT NULL DEFAULT 0;
+
+-- Add custom_subtree_root field to the table asset_groups. This optional field
+-- is used to store the root of the custom subtree for the asset group. The
+-- custom subtree, if provided, represents a subtree of the final tapscript
+-- tree.
+ALTER TABLE asset_groups ADD COLUMN custom_subtree_root BLOB;
+
+-- We're going to recreate key_group_info_view to include the new columns.
+-- Therefore, we need to drop the existing view.
+DROP VIEW IF EXISTS key_group_info_view;
+
+-- Recreate the key_group_info_view to include the new columns.
+--
+-- This view is used to perform a series of joins that allow us to extract
+-- the group key information, as well as the group sigs for the series of
+-- assets we care about. We obtain only the assets found in the batch
+-- above, with the WHERE query at the bottom.
```

Comment:
This is copy pasta from the original migration file. The comment `assets found in the batch above` doesn't make sense here.

```suggestion
-- assets we care about. We obtain the assets with the WHERE query at the
-- bottom.
```